src/config.js

```diff
@@ -1,11 +1,17 @@
+import {inject, Factory} from 'aurelia-dependency-injection';
 import {HttpClient} from 'aurelia-fetch-client';
 import {Rest} from './rest';
 import extend from 'extend';
 
+@inject(Factory.of(HttpClient))
 export class Config {
   endpoints       = {};
   defaultEndpoint = null;
+  httpClientFactory = null;
 
+  constructor(httpClientFactory: Factory) {
```

Comment:
alas, @RWOverdijk doesn't like adding Types. so, you need to kick the : Factory. the dts builder will add ?:any there automatically instead